protobuf/src/main/protobuf/TestProto2.proto

```diff
@@ -0,0 +1,61 @@
+syntax = "proto2";
```

Comment:
These are for testing, should be in `src/test`. Not sure if the protobuf plugin supports that. Or you might have to split them into separate projects like in https://github.com/nevillelyh/protobuf-generic

Reply:
plugin doesn't seem to support it, I made a downstream project protobufTest for them